### ISSUE TITLE
Lower smartquotes mess up highlighting

### ISSUE THREAD

**Issue by zoffixznet** [perl6/atom-language-perl6#48]
### Example Code
```perl6
my $s = sub ($a, $b) { $a + $b };
say $s.WHAT;        # (Sub) 
say $s(2, 5);       # 7 
 
sub postfix:<♥>($a){ say „I love $a!“ }
42♥;
# OUTPUT«I love 42!␤»
sub postcircumfix:<⸨ ⸩>(Positional $a, Whatever){ say $a[0], '…', $a[*-1] }
[1,2,3,4]⸨*⸩;
# OUTPUT«1…4␤»
constant term:<♥> = "♥"; # We don't want to quote "love", do we?
sub circumfix:<α ω>($a){ say „$a is the beginning and the end.“ };
α♥ω;
# OUTPUT«♥ is the beginning and the end.␤»
```
### Picture [optional]
![untitled](https://cloud.githubusercontent.com/assets/5747918/22401581/a1b43478-e5a7-11e6-90bc-907389fcfb26.png)

### Leave this in. For internal use.
- [x] Fixed in Master
- [x] Fixed in Release
- [ ] Has Tests
- [ ] Passes Tests

**Comment by zoffixznet** [comment]
FWIW, these are the combinations we handle: https://github.com/rakudo/rakudo/blob/nom/src/Perl6/Grammar.nqp#L3608-L3616

**Comment by samcv** [comment]
Those all should be easy for me to add, though there were some issues with the reversed quotes:

”this“ < Right double quotationmark + left double quotation mark

**Comment by samcv** [comment]
Fix has been released in v1.14.0, thanks!
